src/calculateRentalCost.js

```diff
@@ -28,7 +28,24 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  let result = days * price;
+
+  if (days < 3) {
+    return result;
+  }
+
+  if (days >= 3 && days < 7) {
```

Comment:
here, `days >= 3` is always true because the previous condition is already omitted